### DISCUSSION TITLE
ETA for New Update?

### DISCUSSION THREAD

**Discussion by magearden** [SwagOtaku/plugin.video.kaito#97] [General]
Sorry if I'm bugging you but is there an ETA of the new update, the add on has been broken for me since the last update, all I get are errors whenever I try to play something from the latest dub section and just anything in general, right now I'm using WNT2 to watch anime in the meantime, don't mean to rush you as I know you've been busy with your own life, just want to know if there is ETA yet of the new update, anyways thank you for working hard on this add on.

**Comment by SwagOtaku** [comment]
https://github.com/SwagOtaku/plugin.video.kaito/discussions/99

**Reply by magearden** [reply]
Thank you, from what I briefly tested everything's seems to be working again 👍 no more errors, again thank you for working hard on the update 😀
